DataFormats/HcalDigi/interface/QIE11DataFrame.h

```diff
@@ -3,13 +3,12 @@
 
 #include "DataFormats/HcalDetId/interface/HcalDetId.h"
 #include "DataFormats/Common/interface/DataFrame.h"
-#include "DataFormats/Common/interface/DataFrameContainer.h"
 #include <ostream>
 
 /** Precision readout digi from QIE11 including TDC information
 
  */
-class QIE11DataFrame : protected edm::DataFrame {
+class QIE11DataFrame {
```

Comment:
do we have any samples with QIE11 (or QIE10) created?
I'm not sure the change in the inheritance is easily supported by the schema evolution.